testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/OID4VCSdJwtIssuingEndpointTest.java

```diff
@@ -61,6 +61,7 @@
 
```

Comment:
IMO, it might be better to write test codes without this protocol mapper, "iat" and "nbf" claims (namely the original test codes).

Reply:
Now i have tests run with and without iat.

Reply:
Addressed with task: https://github.com/keycloak/keycloak/issues/31674